### PR TITLE
Fixes #4787

### LINKED ISSUE #4787: Bluespace crystals can be used on z-level 2
Allows you to get to centcomm using the salvage ship, and from centcomm to many other places (wizard's den, syndie base etc)

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

File: code/modules/telesci/bscrystal.dm [hunk 1]
```diff
@@ -15,9 +15,16 @@
 	pixel_y = rand(-5, 5)
 
 /obj/item/bluespace_crystal/attack_self(var/mob/user)
-	blink_mob(user)
+	var/turf/T = get_turf(src)
+	var/datum/zLevel/path = map.zLevels[T.z]
```
Comment (ComicIronic) [inline code/modules/telesci/bscrystal.dm]: It's not actually a path.
Once the round starts, they're all created datums.

File: code/modules/telesci/bscrystal.dm [hunk 2]
```diff
@@ -25,7 +32,10 @@
 
 /obj/item/bluespace_crystal/throw_impact(atom/hit_atom)
 	..()
-	if(isliving(hit_atom))
+	var/turf/T = get_turf(src)
```
Comment (ComicIronic) [inline code/modules/telesci/bscrystal.dm]: You might want to compress this into a get_z_level() proc or something, might be helpful.

File: code/modules/telesci/bscrystal.dm (hunk 1 above)
Comment (PJB3005) [inline code/modules/telesci/bscrystal.dm]: HA, yes, but the original PATHS in the list never get removed, so a zlevels list(/z1, /z2) ends up list(/z1, /z2, /z1 datum, /z2 datum)

**Comment by clusterfack** [comment]
Is this still fine despite this? https://github.com/d3athrow/vgstation13/commit/942e8820fbabfa066b7a5ef9d83c53a6d6f7ec3f

**Comment by PJB3005** [comment]
Yes it is, after comic made that commit I amended the commit

**Comment by clusterfack** [comment]
Thanks

**Pull request merged**